_episodes/47.md

```diff
@@ -0,0 +1,139 @@
+
+* Experimental support for tracing using OpenTelemetry.
+* Support for Databricks 12.2 LTS in Delta Lake connector.
+* Support for fault-tolerant execution in Redshift connector.
+* `sequence` table function.
+
+## Introducing the two new Trino maintainers
+
+Manfred should hardly need an introduction to Trino Community broadcast viewers,
+as he's been around and hosting episodes for a while. But in the background, 
+he's also been quietly working on docs, the website, and a wide variety of other
+initiatives in the Trino.
+
+James should also be familiar to anyone who has contributed on Trino. Iconically
+rocking a GitHub avatar of the face of Bob Ross, it's hard to miss when he shows
+up on a pull request. And working on Trino as part of AWS Athena, he's been a
+major engineering contributor for the last several years, with 262 commits
+under his belt and more on the way.
+
+## What is a maintainer?
+
+If you don't go clicking around on the Trino website fanatically trying to find
+everything you can possibly read about the project, there's a chance you've
+never bumped into our [roles](https://trino.io/development/roles.html) page,
+which highlights how Trino is governed. To quote that page:
+
+In Trino, maintainer is an active role. A maintainer is responsible for merging
+code only after ensuring it has been reviewed thoroughly and aligns with the
+Trino vision and guidelines. In addition to merging code, a maintainer actively
+participates in discussions and reviews. Being a maintainer does not grant
+additional rights in the project to make changes, set direction, or anything
+else that does not align with the direction of the project. Instead, a
+maintainer is expected to bring these to the project participants as needed to
+gain consensus. The maintainer role is for an individual, so if a maintainer
+changes employers, the role is retained. However, if a maintainer is no longer
+actively involved in the project, their maintainer status will be reviewed.
+
+Or, in normal speech, a maintainer is a trusted individual with merge rights.
+But with great power comes great responsibility, higher standards, and an
+expectation to be an active steward of the Trino project. It's not easy to
+become a maintainer - prior to Manfred and James, it had been over a year since
+the most recent maintainer was appointed. The high bar of activity, quality, and
+attitude is not trivial by any stretch, and so we're excited to talk to them
+about the role, how they got here, and what they're looking forward to for the
+future of Trino.
+
+## The path to becoming a maintainer
+
+### Manfred
+
+* When did you first start working on Trino?
+* What's your proudest contribution to the project?
+* Have a funny story you've wanted to share to the world?
+
+### James
+
+* When did you first start working on Trino?
+* What's your proudest contribution to the project?
+* Have a funny story you've wanted to share to the world?
```

Comment:
Sadly, I think I'm coming up empty on "funny stories", hopefully I can dodge this question :)